java/examples/binary/README.md

```diff
@@ -0,0 +1,14 @@
+# s2i-java-binary-example
+
+This S2I example illustrates how to build a container image for a previously existing Java binary JAR. 
+In this example, one is copied from ../maven/target; but typically you instead would have it downloaded from somewhere.
+
+If you have a project with Java sources, you are probably more interested in using the [Maven](../maven/) or [Gradle](../gradle/) examples, instead.
```

Comment:
I guess the gradle example is still pending (until the respective PR is merged) ?